src/channel.c

```diff
@@ -1765,12 +1765,9 @@ ssize_t _libssh2_channel_read(LIBSSH2_CHANNEL *channel, int stream_id,
                               char *buf, size_t buflen)
 {
     LIBSSH2_SESSION *session = channel->session;
+    LIBSSH2_PACKET *packet, *next_packet;
     int rc;
-    int bytes_read = 0;
-    int bytes_want;
-    int unlink_packet;
-    LIBSSH2_PACKET *read_packet;
-    LIBSSH2_PACKET *read_next;
+    size_t total = 0;
```

Comment:
The previous name was better because it described exactly what it was. `total` could be any number of things